wasm/generateInterface.js

```diff
@@ -223,13 +223,33 @@ function generateManifest (interfaceManifest, opts) {
         // the wasm memory offset is a new item on the EVM stack
         spOffset++
         call += `(i32.add (get_global $sp) (i32.const ${spOffset * 32}))`
+      } else if (input === 'i64' && opcode === 'CALL') {
+        // i64 param for CALL is the gas
+        // add 2300 gas subsidy
+        // for now this only works if the gas is a 64-bit value
+        // TODO: use 256-bit arithmetic
+        /*
```

Comment:
Why is this kept as a comment?

Reply:
it will be relevant when we simplify the EEI and move the ugly gas logic here into evm2wasm